configure.ac

```diff
@@ -62,10 +50,10 @@ AS_IF([test "x$LIBBSD_NEEDED" = "xyes"], [
         [AC_MSG_ERROR([BSD functions not found, libbsd is required])])
 ])
 
-AC_SUBST([LIBS], ["$X11_LIBS $XCOMPOSITE_LIBS $XEXT_LIBS $XFIXES_LIBS \
-    $XINERAMA_LIBS $IMLIB2_LIBS $LIBBSD_LIBS $LIBS"])
-AC_SUBST([CPPFLAGS], ["$X11_CFLAGS $XCOMPOSITE_CFLAGS $XEXT_CFLAGS \
-    $XFIXES_CFLAGS $XINERAMA_CFLAGS $IMLIB2_CFLAGS $LIBBSD_CFLAGS $CPPFLAGS"])
+AC_SUBST([LIBS], ["$SCROT_DEPS_LIBS"])
+AC_SUBST([CPPFLAGS], ["$SCROT_DEPS_CFLAGS"])
+AC_SUBST([CFLAGS], ["$CFLAGS $SCROT_FLAGS"])
+AC_SUBST([LDFLAGS], ["$LDFLAGS $SCROT_FLAGS"])
```

Comment:
This should remove autoconf-archive dependency. I tested both `./configure` without any CFLAGS as well as `./configure CFLAGS="-std=c99"` and both seem to work as expected.

@guijan any downside to this?

Reply:
Yes, the autoconf-archive AX_APPEND_*_FLAGS macro checks if the compiler supports the flags. That would break a compiler that doesn't support e.g. `-flto`.

Reply:
Added a garbage flag to test it out on `master`, looks like it just removes *all* the flags if a single one of them isn't accepted/supported. I guess that's still not too bad. 

I'll see if there's an easy way to do this without autoconf-archive, otherwise I'll add it back.